### PR TITLE
enable job tests in integration tests

### PR DESCRIPTION
**Pull request by abyesilyurt**
## Description
Please include a summary of the change, the motivation, and any additional context that will help others understand your PR. If it closes one or more open issues, [please tag them as described here](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -689,7 +689,7 @@ allowlist_externals =
 setenv =
     NODE_PORT = {env:NODE_PORT:9082}
     GITHUB_CI = {env:GITHUB_CI:false}
-    PYTEST_MODULES = {env:PYTEST_MODULES:frontend network container_workload}
+    PYTEST_MODULES = {env:PYTEST_MODULES:frontend network container_workload local_node}
```
Comment (rasswanth-s) [inline tox.ini]: A small note, Local Node should be not part of k8s
it is already part of 
syft.test.integration workflow

**Review by rasswanth-s — APPROVED** [review]
Well Done @abyesilyurt

**Comment by rasswanth-s** [comment]
This test_job_restart flakes on my machine after repeated runs
Screenshot 2024-05-15 at 3 15 40 PM

**Comment by abyesilyurt** [comment]
I can't reproduce the flakiness atm, will revert this PR if I observe any flakiness on CI.

**Pull request merged**